interface/src/components/App/AppBar.tsx

```diff
@@ -281,7 +281,7 @@ export function AppBarComponent(props: any): any {
                         </div>
                     </Box>
                     <Box className="main-menu--right d-flex">
-                        {dimensions.width > 768 && chain.chainId == "137" ?
+                        {(dimensions.width > 768 && (chain.chainId == "137"|| chain.chainId == "250")) ?
```

Comment:
i suggest to create a list with the chainIDs, otherwise we will have a long condition here.